quarkus-opentelemetry-exporter-sentry/runtime/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.otel.propagators=tracecontext,baggage,sentry
```

Comment:
Putting the `application.properties` here means it will be distributed along with the extension. Are you sure it is what you want?

Reply:
I wonder if there isn't a BuildItem to avoid that

Reply:
Not that I'm aware of, but the `ExternalOtelExporterBuildItem` could be updated in a way that could transport that info.

Reply:
I thought a bit more about it... It is pointless to use a build item for this because we are not setting any API ourselves... We are serving as a passthrough to those properties because they are consumed inside the OpenTelemetry SDK.